### DISCUSSION TITLE
Symfony bundle for Tabler integration

### DISCUSSION THREAD

**Discussion by kevinpapst** [tabler/tabler#954] [Show and tell]
I am using Tabler in one of my open-source projects (build on top of the PHP framework Symfony) and thought about a way to share the work that I am doing, in order to simplify integration for the next developer.

So I started working on a Symfony bundle that allows to integrate the Tabler theme into an Symfony application. 
The integration is simple if you are starting a new project, if you have an existing application it might get "messy". When I migrated my application from another well known (but sold out and outdated) open source admin theme to tabler, most of the work was coming from the switch between Bootstrap 3 to 5.

I am really grateful that this theme exists and want to give back to the community - so here is the link to the repository: https://github.com/kevinpapst/TablerBundle
If you use Symfony for your application, you are welcome to try the bundle..

Keep up the great work @codecalm and everyone else supporting this project!

Here is a demo application: https://github.com/kevinpapst/TablerBundle-Demo that serves as living example

**Comment by mlejeunedev** [comment]


**Reply by kevinpapst** [reply]
Please post in the correct repo: https://github.com/kevinpapst/TablerBundle
